### ISSUE TITLE
Copy and paste clipboard issue while javascript is active.

### ISSUE THREAD

**Issue by voxel-minetest** [Ionaru/easy-markdown-editor#321]
Hi. 
I was redirected from here: https://github.com/minetest/contentdb/issues/292

**Comment by Ionaru** [comment]
As far as I'm aware, pasting text using the middle-mouse is not standard browser behaviour. Usually, middle-mouse click is reserved for scrolling or opening links in a new tab/window. Some Linux distros allow it, but it is far from universal.

EasyMDE does not handle any copy-paste events, these are sent down to the CodeMirror editor.

I found these reported issues that discuss the topic:
- https://github.com/codemirror/CodeMirror/issues/931 (Links to an open Firefox bug)
- https://github.com/codemirror/CodeMirror/issues/4900 (Links to a Chrome bug, but it is not public)

You should have a look on https://codemirror.net/ (current editor in use) or https://codemirror.net/6/ (future version) to see if it works there.

tldr: there's nothing that can be changed about it in EasyMDE.

**Comment by voxel-minetest** [comment]
> As far as I'm aware, pasting text using the middle-mouse is not standard browser behaviour. Usually, middle-mouse click is reserved for scrolling or opening links in a new tab/window. Some Linux distros allow it, but it is far from universal.

Ehm.. no that is the default behaviour of the primary clipboard in probably all distributions:
https://specifications.freedesktop.org/clipboards-spec/clipboards-latest.txt 

Also I have never had this issue on any other editor, no matter with js enabled or not, while using firefox and I use this thing for nearly 15 years now.

> tldr: there's nothing that can be changed about it in EasyMDE.

I see.
Thx anyway.
